### ISSUE TITLE
[print css] Fix unwanted page-breaks in inner content

### ISSUE THREAD

**Issue by plutonik-a** [eXistSolutions/hsg-shell#227]
Reported in https://github.com/eXistSolutions/hsg-shell/pull/226#issuecomment-219835031
Related https://github.com/eXistSolutions/hsg-shell/issues/219
[http://localhost:8080/exist/apps/hsg-shell/departmenthistory/people/jefferson-thomas](http://localhost:8080/exist/apps/hsg-shell/departmenthistory/people/jefferson-thomas)

There is a strange bug, possibly a layout directive coming from ODD, which is overriding my CSS styles for page-breaks, that causes a page break right before the `class="tei-head-*"`, but only when the content is wrapped in class `#content-container`.

<img width="196" alt="bildschirmfoto 2016-05-17 um 21 54 43" src="https://cloud.githubusercontent.com/assets/1157669/15337121/6324cc6e-1c7a-11e6-8f08-2e37dd463c52.png">

**Comment by plutonik-a** [comment]
After further research and attempts of fixing this, I found out, that the bug is depending on the user agent + OS.
I compared the print results of Safari, FF and Chrome. All were slightly different but acceptable, except for Chrome, which I am developing with.
Chrome obviously doesn't care about the rules, that I defined for page-breaks, even though I set e.g. `img { page-break-inside: avoid !important; }` - just look at the screenshot below, where the page-break is done right in the middle of the image!

<img width="215" alt="bildschirmfoto 2016-05-18 um 12 05 55" src="https://cloud.githubusercontent.com/assets/1157669/15355241/d5ec009c-1cf1-11e6-9df5-76dc07a797d4.png">

<img width="168" alt="bildschirmfoto 2016-05-18 um 12 08 09" src="https://cloud.githubusercontent.com/assets/1157669/15355247/dd336f02-1cf1-11e6-87dd-a392fde61e84.png">

<img width="199" alt="bildschirmfoto 2016-05-18 um 12 02 21" src="https://cloud.githubusercontent.com/assets/1157669/15355269/fd415a66-1cf1-11e6-9694-0a4b8604df75.png">
